src/@hooks/usePrivacyMetadata.ts

```diff
@@ -13,6 +14,9 @@ export interface UsePrivacyMetadata {
   }[]
 }
 
-export function usePrivacyMetadata(): UsePrivacyMetadata {
+export function usePrivacyMetadata(isCookie: boolean): UsePrivacyMetadata {
+  if (isCookie) {
+    return { ...cookieContent }
+  }
   return { ...privacyContent }
 }
```

Comment:
todo: Right now you’re calling it `usePrivacyMetadata` but driving its behavior with an `isCookie` boolean. In practice this makes the hook name misleading, what you’re really toggling between is “cookie policy” vs. “privacy policy” metadata. 
Split into two hooks:

```suggestion
export function usePrivacyPolicies(): UsePolicyMetadata {
  return { ...privacyContent }
}

export function useCookiePolicies():    UsePolicyMetadata {
  return { ...cookieContent }
}
```